src/Goerli-DssSpell.sol

```diff
@@ -50,79 +52,119 @@ contract DssSpellAction is DssAction {
-        // SG Forge OFH (RWA008-A) legal update doc
-        _updateDoc("RWA008-A", "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy");
-        // HVB (RWA009-A) legal update doc
-        _updateDoc("RWA009-A", "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT");
-
-        // RWA007-A autoline changes:
-        // - bump oralce price to 500m
-        // - increase DC to 500m
-        // - increase autoline `gap` to 100m
-        //
-        // https://vote.makerdao.com/polling/QmSfMtTM#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RWA007-A",
-            500 * MILLION,
-            100 * MILLION,
-            1 weeks
-        );
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
-            "RWA007-A",
-             500 * MILLION * WAD
-        );
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // RETH-A autoline changes:
-        // - line 5m
-        // - gap 3m
-        // - ttl 8h
-        //
-        //  https://vote.makerdao.com/polling/QmfMswF2#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RETH-A",
-            5 * MILLION,
-            3 * MILLION,
-            8 hours
-        );
-
-        // -------------------- Changelog Update & housekeeping ---------------------
-
-        // - Change "RWA007_A_INPUT_CONDUIT_URN" to "RWA007_A_INPUT_CONDUIT"
-        // - Change "RWA007_A_INPUT_CONDUIT_JAR" to "RWA007_A_JAR_INPUT_CONDUIT"
-        //
-        DssExecLib.setChangelogAddress("RWA007_A_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_URN"));
-        DssExecLib.setChangelogAddress("RWA007_A_JAR_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_JAR"));
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_URN");
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_JAR");
-
-        // Bump version
-        DssExecLib.setChangelogVersion("1.14.5");
+        // ----------------- Offboard GUSD-A, USDC-A and USDP-A -----------------
+        // https://vote.makerdao.com/polling/QmZbsHqu#poll-detail
+        {
+            bytes32 _ilk  = bytes32("GUSD-A");
+            address _clip = DssExecLib.getChangelogAddress("MCD_CLIP_GUSD_A");
+            //
+            // Enable liquidations for GUSD-A
+            // Note: ClipperMom cannot circuit-break on a DS-Value but we're adding
+            //       the rely for consistency with other collaterals and in case the PIP
+            //       changes to an OSM.
+            DssExecLib.authorize(_clip, DssExecLib.clipperMom());
+            DssExecLib.setValue(_clip, "stopped", 0);
+            // Use Abacus/LinearDecrease
+            DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_GUSD_A);
+            // Set Liquidation Penalty to 0
+            DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
+            // Set Liquidation Ratio to 150%
+            DssExecLib.setIlkLiquidationRatio(_ilk, 15000);
```

Comment:
Apologies if this is a stupid question but where is this implied to be something that changes this spell? I'm assuming that the change to `buf` is what causes this somehow, but since it isn't in the checklist nor the sheet I wanted to be sure.